### PR TITLE
Todolist features

### PR DESCRIPTION
**Pull request by biruk-bereka**
## In this PR I introduced different features for todolist project:
   - User can add, edit and delete a task.
   - Every task is saved in local storage.
   - Webpack used.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,154 @@
...
+- Function for "Clear all completed" button implemented
+- Add event listener to checkboxes of the list and mark the object as complete and incomplete depending on the checkbox.
+- Add interactivity for description and checkbox
+- Store the updated array of items in the local storage.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
```
Comment (jugosack) [inline README.md:74]: ### General comments
The link for GitHub pages is missing. Also, there should be added link that follow https:// concept.

File: src/index.html [hunk 2]
```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
```
Comment (jugosack) [inline src/index.html:1]: [HTML and CSS best practices](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md).

- HTML follows best practices.

File: src/index.html [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+        <h3>Today's To Do</h3>
+        <button class="referesh"><i class="fa-solid fa-arrows-rotate"></i></button>  
+      </div>
+      <form action="">
+        <input class="listInput" type="text" placeholder="Add to your list..." />
+        <button class="addList"><i class="fa-solid fa-check"></i></button>
+      </form>
+      <section class="todo-lists">
+         <ul class="lists">
+           <!-- Dynamically generated lists goes here.. -->
```
Comment (jugosack) [inline src/index.html:20]: - Properly added a comment.

File: src/modules/todoList.js [hunk 4]
```diff
@@ -0,0 +1,142 @@
...
+          const statusUpdate = new Status();
+          statusUpdate.status(index + 1, event.target.checked);
+        });
+      });
+    } else {
+      const todoListsWrapper = document.querySelector('.lists');
+      todoListsWrapper.innerHTML = '';
+    }
+  }
+}
```
Comment (jugosack) [inline src/modules/todoList.js]: [JavaScript best practices](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/javascript_best_practices.md)
- This part of the code with functions could be refactored to a separate module and imported where it is needed.

**Review by jugosack — CHANGES_REQUESTED** [review]
Hi @biruk-bereka ,
Good job so far 👍🏼
Your code is great.
There are some issues that you still need to work on to make your code clean and follow JS best practices.
Fell free to contact me for any questions.
Please check the comments below.

File: src/modules/todoList.js (hunk 4 above)
Comment (biruk-bereka) [reply src/modules/todoList.js]: Thank you @jugosack for the comment.

**Review by jugosack — APPROVED** [review]
### Status: Approved
@biruk-bereka:
Nothing else to say than... it is time to merge 💯 
Well done 🥇

**Pull request merged**